src/app/heroes/heroes.component.ts

```diff
@@ -29,12 +29,20 @@ export class HeroesComponent implements OnInit {
 
     this.heroes.items.push(hero);
     this.http.post(this.heroes._links.self.href, hero)
-      .subscribe(undefined, undefined, () => this.view.reload());
+      .subscribe({
+        next: undefined,
```

Comment:
Got it, would just pass the arrow function as an argument then (although it becomes _next_ parameter instead of _complete_, but it doesn't affect the behavior, so i guess it's ok).